src/utils/constants.ts

```diff
@@ -52,6 +52,11 @@ export enum FEEDBACK_TAGS_ENUM {
   NOT_USEFUL = 'not useful',
 }
 
+export enum RESOURCE_TYPE_ENUM {
+  SHORT = 'short',
```

Comment:
Might be worth making these more verbose, short_video, conversation_audio?

Reply:
Yeah that could work. You're call!